config/locales/views/user_mailer/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  user_mailer:
+    new_rent:
+      title: "New rent created"
+      creation_date: "Creation date: "
+      book: "Book: "
```

Comment:
This translation should not be here, you should make a translation file of the model book, check this link:
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

Reply:
Then where you used this translation, you should use the Model one, because later if the model translation change, you only change it in one place.